Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.tsx

```diff
@@ -120,9 +120,11 @@ const SubmissionGridRow = ({ submission }: ISubmissionGridRowProps) => {
                     />
                 </div>
                 <div className={styles.dateAndUsernameContainer}>
-                    <span>{formatDate(createdOn)}</span>
-                    <span>by</span>
-                    <span>{username}</span>
+                    <span>
```

Comment:
This way there is no space between created on, by and the username. You can put empty spaces between them like this: 
                      <span>
                        {formatDate(createdOn)}
                        {' '}
                        by
                        {' '}
                        {username}
                    </span>

Reply:
ESLint doesn't want them. There is the following error: Curly braces are unnecessary here.